README.md

```diff
@@ -47,22 +47,21 @@ For Level Zero memory provider tests:
 ### Linux
 
 Executable and binaries will be in **build/bin**
```

Comment:
pls add a full stop at the end of this line - it's all displayed as one sentence and it reads weird

// preview: https://github.com/PatKamin/unified-memory-framework/tree/simplify-build-steps?tab=readme-ov-file#linux

Reply:
Done